src/main/webapp/resources/js/pages/dashboard/components/RecentActivity.jsx

```diff
@@ -0,0 +1,59 @@
+import { Button, Space, Typography } from "antd";
+import React from "react";
+import { grey1 } from "../../../styles/colors";
+import { SPACE_MD } from "../../../styles/spacing";
+import { RecentActivityAllProjects } from "./RecentActivityAllProjects";
+import { RecentActivityUserProjects } from "./RecentActivityUserProjects";
+
+const isAdmin = window.TL._USER.systemRole === "ROLE_ADMIN";
```

Comment:
There is an `isAdamin` function in `role-utilities.js` that handles tihs better.  I still hate that it checkes the `window.TL` variable, we need to eventually get rid of that.

Reply:
Yeah we need to figure it out and change it! Updated in 0042a0ad2e7a59b91ddeb0ff9efe5adaa46c6c25.